.github/mergeable.yml

```diff
@@ -0,0 +1,269 @@
+    pass:
+      - do: labels
+        add: 'status: needs work'
+        delete: 'status: needs review'
+      - do: comment
+        payload:
+          body: 'As the reviewer has requested changes, we have added "status: needs work" and have removed "status: needs review" changes.'
+
+  # Let's add "maintainers" team as reviewer once, 'optional: review requested' label is added.
+  - when: pull_request.*
+    name: 'Request mainatainer for review'
+    filter:
+      - do: payload
+          review:
+            state:
+              must_include:
+                regex: 'review_requested'
+    validate:
+      - do: label
+        begins_with:
+          match: 'optional: review_requested'
+    pass:
+      - do: request_review
+        teams: ['maintainers']
+      - do: comment
+          payload:
+            body: 'As the @author has requested review from maintainers, we have added @goalgorilla/maintainers reviewers.'
+
+  # We want to make sure that the cherry-pick was completed after the PR was merged.
+  # This check will run after 24 hours of PR merging.
+  # see: https://mergeable.readthedocs.io/en/latest/recipes.html?highlight=stale#check-stale-pr-and-issues
+  - when: schedule.repository
+    name: 'Check backport'
+    type: pull_request
+    time_constraint:
+      time_zone: 'Europe/Amsterdamn'
+      hours_between: [ '9', '17' ]
+      days_of_week: [ 'Mon', 'Tue', 'Wed', 'Thu', 'Fri' ]
+    filter:
+      and:
+        - do: payload
+          pull_request:
+            merged:
+              boolean:
+                match: true
+        - do: payload
+          pull_request:
+            labels:
+              must_include:
+                regex: 'status: needs backport'
+                key: 'name'
+    validate:
+      and:
+        - do: stale
+          days: 1
+          type: pull_request
+        - do: label
+          must_include:
+            regex: 'verified-cherry-picks'
+            message: 'Cherry pick flag is required'
+    pass:
+      - do: comment
+        payload:
+          body: 'Great job! Please relax. The PR is complete.'
+    fail:
+      - do: comment
+        payload:
+          body: >
+            Hurray! Your PR was merged, yesteday. We see that PR has 'status: needs backport' label. We, 'pretty please' you to backport (cherry-pick)
+            the changes to mentioned milestone.
+      - do: checks
+        status: 'action_required'
+
+  # We want to remind about PRs older than 30 days
+  # See: https://mergeable.readthedocs.io/en/latest/validators/stale.html
+  - when: schedule.repository
+    name: "Validate stale PR"
+    type: pull_request
+    time_constraint:
+      time_zone: 'Europe/Amsterdamn'
+      hours_between: [ '9', '17' ]
+      days_of_week: [ 'Mon', 'Tue', 'Wed', 'Thu', 'Fri' ]
+    validate:
+        - do: stale
+          days: 30
+          type: pull_request
+    pass:
+      - do: comment
+        payload:
+          body: 'This is old. Is it still relevant?'
```

Comment:
This is a cool addition! But we might be able to provide a little better experience.

For example what happens if one of the teams didn't do their review? Or it became stale because Nucleus forgot something? 

Instead alternatives like:
We see this Pull Request is untouched for 30 days, do you think this PR can be closed? Let me ping @goalgorilla/maintainers anyway so they might be able to assist :)

Reply:
@ronaldtebrake awesome suggestion. I will do it.